packages/eslint-config-audius/src/index.js

```diff
@@ -0,0 +1,107 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: [
+    "standard",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:jest/recommended",
+    "plugin:react/recommended",
+    "plugin:import/typescript",
+    "prettier",
+    "prettier-standard/prettier-file",
+  ],
+  globals: {
+    Atomics: "readonly",
+    SharedArrayBuffer: "readonly",
+  },
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: "module",
+  },
+  plugins: ["react", "react-hooks", "@typescript-eslint", "jest", "import"],
```

Comment:
If we wanna apply this same config to our services too, it might make sense so split out the react stuff into `audius-react` or something. But we can do that in a future pr!